test/setupJest.ts

```diff
@@ -0,0 +1 @@
+import 'whatwg-fetch'
```

Comment:
seems whatwg-fetch is not really been used? just for jest to call spyOn and override implementation. how about providing a fake fetch on nodejs `global` and clean it up when test is finished

```diff
 it('should use fetch api as default fetcher', async () => {
    const users = [{ name: 'bob' }, { name: 'sue' }]
+    global['fetch'] = () => Promise.resolve()
    const mockFetch = body =>
      Promise.resolve({ json: () => Promise.resolve(body) } as any)
    ...
    expect(container.firstChild.textContent).toMatchInlineSnapshot(
      `"hello, bob and sue"`
    )
+    delete global['fetch']
  })
```

Reply:
yes this is much better, thanks!